src/Microsoft.AspNet.Razor/Parser/HtmlMarkupParser.Block.cs

```diff
@@ -558,19 +558,33 @@ private void AttributeValue(HtmlSymbolType quote)
 
             if (At(HtmlSymbolType.Transition))
             {
-                var valueStart = CurrentLocation;
-                PutCurrentBack();
+                if (NextIs(HtmlSymbolType.Transition))
+                {
+                    // Render a single "@" in place of "@@".
+                    var transition = CurrentSymbol;
```

Comment:
Hmm I think this is an outstanding issue but I believe if you type `@@` anywhere in the document and you continue typing, such as: `@@` => `@DateT@` I don't belive you'll get IntelliSense because the `AcceptedCharacters` is `Any`.

To do this right from an editor perspective we should change the `AcceptedCharacters` for both `@` symbols to be `AcceptedCharacters.None`.

Aka:
```C#
Output(SpanKind.Markup);
AcceptAndMoveNext();
Span.CodeGenerator = SpanCodeGenerator.Null;
Output(SpanKind.Markup, AcceptedCharacters.None);
AcceptAndMoveNext();
Output(SpanKind.Markup, AcceptedCharacters.None);
```

This way we're outputting the second `@` symbol early with a different `AcceptedCharacters` value.  Feel free to update the other document level escaping `@` bits or file an issue.